<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-10T09:16:17 PST" name="Suite" finished-at="2023-02-10T09:17:05 PST" duration-ms="47720">
    <groups>
    </groups>
    <test started-at="2023-02-10T09:16:17 PST" name="chromeTests" finished-at="2023-02-10T09:17:05 PST" duration-ms="47720">
      <class name="com.jan23.tests.TekarchAutomation">
        <test-method is-config="true" signature="setUpBeforeTest()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:18 PST" name="setUpBeforeTest" finished-at="2023-02-10T09:16:19 PST" duration-ms="1685" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeTest -->
        <test-method is-config="true" signature="setUpBeforeTestMethod(java.lang.String,java.lang.reflect.Method)[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:20 PST" name="setUpBeforeTestMethod" finished-at="2023-02-10T09:16:29 PST" duration-ms="9913" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void com.jan23.tests.TekarchAutomation.Student_registration_Form() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeTestMethod -->
        <test-method signature="Student_registration_Form()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:29 PST" name="Student_registration_Form" finished-at="2023-02-10T09:16:38 PST" duration-ms="8870" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at com.jan23.tests.TekarchAutomation.Student_registration_Form(TekarchAutomation.java:130)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:208)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:159)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:87)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:95)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Student_registration_Form -->
        <test-method is-config="true" signature="teardownAfterTestMethod()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:39 PST" name="teardownAfterTestMethod" finished-at="2023-02-10T09:16:39 PST" duration-ms="297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeTestMethod(java.lang.String,java.lang.reflect.Method)[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:39 PST" name="setUpBeforeTestMethod" finished-at="2023-02-10T09:16:46 PST" duration-ms="6978" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void com.jan23.tests.TekarchAutomation.error_loginToFirebase() throws java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeTestMethod -->
        <test-method signature="error_loginToFirebase()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:46 PST" name="error_loginToFirebase" finished-at="2023-02-10T09:16:54 PST" duration-ms="7854" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- error_loginToFirebase -->
        <test-method is-config="true" signature="teardownAfterTestMethod()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:54 PST" name="teardownAfterTestMethod" finished-at="2023-02-10T09:16:54 PST" duration-ms="268" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeTestMethod(java.lang.String,java.lang.reflect.Method)[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:16:54 PST" name="setUpBeforeTestMethod" finished-at="2023-02-10T09:17:01 PST" duration-ms="7339" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void com.jan23.tests.TekarchAutomation.loginToFirebase() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeTestMethod -->
        <test-method signature="loginToFirebase()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:17:01 PST" name="loginToFirebase" finished-at="2023-02-10T09:17:04 PST" duration-ms="2884" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToFirebase -->
        <test-method is-config="true" signature="teardownAfterTestMethod()[pri:0, instance:com.jan23.tests.TekarchAutomation@6b419da]" started-at="2023-02-10T09:17:04 PST" name="teardownAfterTestMethod" finished-at="2023-02-10T09:17:05 PST" duration-ms="481" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownAfterTestMethod -->
      </class> <!-- com.jan23.tests.TekarchAutomation -->
    </test> <!-- chromeTests -->
  </suite> <!-- Suite -->
</testng-results>
